### PR TITLE
Create a page for community translations

### LINKED ISSUE #430: Create a page for community translations
Just a basic page where staff can add links, I think

### PR DESCRIPTION
**Pull request by wioux**
Fixes #430. Also narrows the margins on /credits and /feedback.

### CODE REVIEW

**Comment by wioux** [comment]
I'm holding off on adding the footer link for now, since the translations page has no content yet.

File: app/controllers/pages_controller.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+class PagesController < ApplicationController
```
Comment (vbrown608) [inline app/controllers/pages_controller.rb]: :+1:

**Comment by vbrown608** [comment]
This needs to be added to the admin interface - you can do that in `config/initializers/active_admin`.

**Comment by wioux** [comment]
> This needs to be added to the admin interface - you can do that in config/initializers/active_admin.

Done!

> (Might be a good time to generalize that to the page model)

Not done. I'm creating an issue for follow up.

**Pull request merged**